### DISCUSSION TITLE
Is the latest 7.0 beta stable?

### DISCUSSION THREAD

**Discussion by anthony-tron** [Comcast/react-data-grid#2936] [Q&A] [answered]
I've been looking for the latest release of react-data-grid and it seems it was something released in 2018 (5.4.x). Then, only alphas, betas, canaries (what are these by the way?) have been published.

Is the latest 7.0-beta stable? What are common caveats?

**Comment by nstepien** [comment, marked as answer]
We use the latest 7.0 beta version and it's been stable for us, we just cannot guarantee API stability until we release 7.0.0 (non-beta). The documentation isn't complete yet either.

**Reply by venugopalkathavate** [reply]
@nstepien Is there a tentative timeline for the release of the stable version?

**Comment by pendarasantiago** [comment]
After I install react-data-grid@7.0.0beta.13 i get the following error.

error - Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/../react/jsx-runtime' imported from /Users/../node_modules/react-data-grid/lib/bundle.js
Did you mean to import react/jsx-runtime.js?

**Reply by abhijeetyou98** [reply]
bash :
npm install react/jsx-runtime


bash: 
rm -rf node_modules
npm install

Change the JSON file a little bit. I hope this will work for you.

Or you can reinstall everything from the start.
